pkgs/by-name/et/ethercat/package.nix

```diff
@@ -57,6 +67,39 @@ stdenv.mkDerivation (finalAttrs: {
 
   passthru.updateScript = gitUpdater { };
 
+  # See: https://gitlab.com/etherlab.org/ethercat/-/blob/6e8119b95563ba955954a68e5e2f4f3f861ac72e/.gitlab-ci.yml#L117
+  postPatch = ''
+    git show -s --format="\def\revision{%h}\def\gitversion{%(describe)}\def\gittag{%(describe:abbrev=0)}\def\gitauthor{%an}\def\isodate#1-#2-#3x{\day=#3 \month=#2 \year=#1}\isodate %csx" HEAD > documentation/git.tex
+  '';
+
+  postInstall = ''
+    mkdir -p $doc
+
+    echo "Build Doxygen docs"
+    make doc
+    cp -r doxygen-output/html $doc/html
+
+    echo "Build Doxygen LaTeX docs"
+    make -C doxygen-output/latex
+    cp -r doxygen-output/latex/refman.pdf $doc/ethercat_ref.pdf
+
+    echo "Build LateX manual"
+    mkdir -p documentation/external
+    make -C documentation
+    make -C documentation index
+    make -C documentation
+
+    cp documentation/*.pdf $doc
+  '';
+
+  postFixup = ''
+    mv $out/{share,lib,etc} $bin
+    find $out
+    echo
+    find $bin
```

Comment:
```suggestion
```

Reply:
Fixed in 14a1b28727fac1532a2b21269c0f5f0d317e4405